cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -640,6 +640,13 @@ export class Application {
         Assessment.validateReviewFields(this.name, "Application");
     }
 
+    validateInheritedReviewFields(archetypes): void {
```

Comment:
Lets rename the param to archetypesList to make it clear .

validateInheritedReviewFields(archetypesList):